grid_map_cv/CMakeLists.txt

```diff
@@ -43,10 +43,14 @@ add_library(${PROJECT_NAME} SHARED
   src/InpaintFilter.cpp
 )
 
-ament_target_dependencies(${PROJECT_NAME} SYSTEM
+ament_target_dependencies(${PROJECT_NAME} PUBLIC
   ${dependencies}
 )
 
+target_link_libraries(${PROJECT_NAME} PUBLIC
+  grid_map_core::grid_map_core
```

Comment:
How about adding other dependencies like `rclcpp::rclcpp` and removing `ament_target_dependencies`?

Reply:
Yes, with more time, it would be a great idea to do that. Do you have ability to help out?

Reply:
I could split this PR to only touch `grid_map_core`, and then do one package at a time.

Reply:
@Ryanf55 
> Do you have ability to help out?
Yes, I will work on it after this PR is once merged.

Reply:
Here is [the PR](https://github.com/Ryanf55/grid_map/pull/1)